src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/SoftwareEntitlementClient.h

```diff
@@ -54,7 +54,8 @@ class Entitlement
 std::unique_ptr<Entitlement> GetEntitlement(
     std::string url,
     const std::string& entitlement_token,
-    const std::string& requested_entitlement
+    const std::string& requested_entitlement,
+    unsigned int retries = 5
```

Comment:
Do we have any external config/settings for the native library we could use for this? It would be useful to be able to increase this if we find there are still issues without the Arnold guys needing to recompile everything...